plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -928,7 +933,7 @@ private static function _helper_subscription_activate() {
 	public static function activate_helper_subscription( $product_key ) {
 		$subscription = self::get_subscription( $product_key );
 		if ( ! $subscription ) {
-			return false;
+			throw new Exception( __('Subscription not found', 'woocommerce') );
```

Comment:
PhpStorm suggests we add a `@throws Exception` to the function docblock to account for this. (Same with `deactivate_helper_subscription`.) 

It also says that `$activated` is always true when returned on line 993, and suggests we just return `true`.

Reply:
[059d447](https://github.com/woocommerce/woocommerce/pull/40703/commits/059d44760a01980d244d1acb0eb679e64c863b42)